include/Clip.h

```diff
@@ -53,14 +53,17 @@ class LMMS_EXPORT Clip : public Model, public JournallingObject
 	mapPropertyFromModel(bool,isMuted,setMuted,m_mutedModel);
 	mapPropertyFromModel(bool,isSolo,setSolo,m_soloModel);
 public:
-	Clip( Track * track );
+	Clip();
+	Clip(const Clip& clip);
```

Comment:
Maybe it would be better to have a `clone` method rather than a public copy constructor in this polymorphic base class?